cmd/mattermost/commands/user.go

```diff
@@ -368,6 +379,52 @@ func userCreateCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func usersToBots(command *cobra.Command, args []string, a *app.App) error {
+	if len(args) < 1 {
+		return errors.New("Expected at least one argument. See help text for details.")
+	}
+
+	users := getUsersFromUserArgs(a, args)
+	for i, user := range users {
+		if user == nil {
+			CommandPrintErrorln(fmt.Errorf("Unable to find user \"%s\"", args[i]))
+			continue
+		}
+
+		bot, err := a.ConvertUserToBot(user)
+		if err != nil {
+			CommandPrintErrorln(err.Error())
+			continue
+		}
+
+		CommandPrettyPrintln("ownerId: " + bot.OwnerId)
```

Comment:
I guess this text here was for debugging purpose, right?

Reply:
If you are talking about `ownerId` specifically, I think it is a good-to-have but I am open to remove it. Let me know what you think.

Reply:
Nope, I was talking about the whole set of prints there, imagine that I convert 3 users to bots, is just a bunch of lines with a clear relation between them. I would be open to something more simple here like one line per migrated user with all the relevant information, probably just something like "User X is converted to bot successfully" or something like that.